homeassistant/components/zone.py

```diff
@@ -44,6 +44,7 @@ def active_zone(hass, latitude, longitude, radius=0):
         within_zone = zone_dist - radius < zone.attributes[ATTR_RADIUS]
         closer_zone = closest is None or zone_dist < min_dist
         smaller_zone = (zone_dist == min_dist and
+                        zone.attributes[ATTR_RADIUS] != 0 and
```

Comment:
Perfect I'll delete this hack!

Still good to have 'consider home' for beacon zones.